clients/lsp-clojure.el

```diff
@@ -304,5 +304,19 @@ If there are more arguments expected after the line and column numbers."
 
 (lsp-consistency-check lsp-clojure)
 
+;; Cider integration
+
+(declare-function lsp-semantic-tokens--enable "lsp-semantic-tokens")
```

Comment:
Why not require `lsp-semantic-tokens` directly?
Also, the `lsp-semantice-tokens--enabled` is autoloader already, so this declaration may not needed

Reply:
oh good to know, so I don't need to even require? how to suppress the warning?